lib/sorcery.rb

```diff
@@ -1,5 +1,13 @@
 require 'sorcery/version'
 
+if RUBY_VERSION < '2.0.0'
+  warn '[DEPRECATION] Sorcery 0.10.0 will deprecate Ruby 1.9.3 and lower, please upgrade to Ruby 2.0 or higher.'
+end
+
+if Rails::VERSION < '4.0.0'
```

Comment:
You're welcome. I'm glad to hear that helped. Although I haven't checked the details yet, the fixes for Travis on master might help you to solve remaining problems. You should probably see #764 and consider backporting.